manim/stream_starter.py

```diff
@@ -0,0 +1,146 @@
+import code
+import functools
+import os
+import readline
+import rlcompleter
+import subprocess
+
+from colorama import Fore, Style
+
+from . import config, logger
+from .scene.streaming_scene import get_streamer, play_scene
+
+
+__all__ = ["livestream", "stream", "open_client"]
+
+
+globals().update(config.streaming_config)
```

Comment:
Question : what is this for?

Reply:
To make the streaming configurations available in the file

Yeah, yeah, it's best if I did it manually. If that's quite important then I'll change it

Reply:
I see. Maybe @leotrs has something to say about this, as super master config.